### ISSUE TITLE
Educate users to use helper apps to input URLs

### ISSUE THREAD

**Issue by bbinto** [mozilla-mobile/firefox-tv#382]
e.g. into onboarding?

Fire TV users can use the App to input URLs (though we can make this experience better)

Depends on #276

**Comment by Sdaswani** [comment]
Here is an example: https://photos.app.goo.gl/xJy02S1gucyBqT253

**Comment by liuche** [comment]
We need to make sure #276 (autocomplete breaking text input) is fixed before we do this.

**Comment by mcomella** [comment]
Project PushBox is a lightweight send-tab-to-device client from desktop Firefox <-> iOS. It doesn't have the dependencies of all of Sync and authenticates with Firefox Accounts – maybe we can use this to enter URLs quickly on the TV device.

**Comment by mcomella** [comment]
May be solved with custom keyboard: #435

**Comment by athomasmoz** [comment]
Closing as low priority.
